rts/Lua/LuaShaders.cpp

```diff
@@ -986,6 +985,15 @@ int LuaShaders::GetUniformLocation(lua_State* L)
 	return 1;
 }
 
+/*** Get subroutine index.
+ *
+ * @function gl.GetSubroutineIndex
+ *
+ * @param shaderID integer
```

Comment:
new function doc, please review argument types (and whether we already have a type for them apart from `integer`)